<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScriptEditorPageScriptOptions</class>
 <widget class="QWidget" name="ScriptEditorPageScriptOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListWidget" name="listOptions"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="buttonOptionsNew">
         <property name="text">
          <string>New Option</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonOptionsMoveUp">
         <property name="text">
          <string>Move Up</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonOptionsMoveDown">
         <property name="text">
          <string>Move Down</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonOptionsDelete">
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="1">
      <widget class="QComboBox" name="optionType">
       <item>
        <property name="text">
         <string>Enum</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Range</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Option Description</string>
       </property>
       <property name="buddy">
        <cstring>optionDesc</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Option Name</string>
       </property>
       <property name="buddy">
        <cstring>optionName</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="ReachStringPicker" name="optionDesc" native="true"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Option Type</string>
       </property>
       <property name="buddy">
        <cstring>optionType</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="ReachStringPicker" name="optionName" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stack">
     <widget class="QWidget" name="pageEnum">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QListWidget" name="listValues"/>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QPushButton" name="buttonValuesNew">
               <property name="text">
                <string>New Value</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="buttonValuesMoveUp">
               <property name="text">
                <string>Move Up</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="buttonValuesMoveDown">
               <property name="text">
                <string>Move Down</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="buttonValuesDelete">
               <property name="text">
                <string>Delete</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="2" column="1">
            <widget class="ReachStringPicker" name="valueDesc" native="true">
             <property name="toolTip">
              <string>When this value is selected, its description will be shown in place of the option description, unless the value description is zero-length.</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Value</string>
             </property>
             <property name="buddy">
              <cstring>valueValue</cstring>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="ReachStringPicker" name="valueName" native="true"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Value Name</string>
             </property>
             <property name="buddy">
              <cstring>valueName</cstring>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_8">
             <property name="toolTip">
              <string>When this value is selected, its description will be shown in place of the option description, unless the value description is zero-length.</string>
             </property>
             <property name="text">
              <string>Value Description</string>
             </property>
             <property name="buddy">
              <cstring>valueDesc</cstring>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QSpinBox" name="valueValue">
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="minimum">
              <number>-512</number>
             </property>
             <property name="maximum">
              <number>511</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageRange">
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="2" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Range Default</string>
         </property>
         <property name="buddy">
          <cstring>rangeDefault</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Range Maximum</string>
         </property>
         <property name="buddy">
          <cstring>rangeMax</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="rangeMin">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="minimum">
          <number>-512</number>
         </property>
         <property name="maximum">
          <number>511</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Range Minimum</string>
         </property>
         <property name="buddy">
          <cstring>rangeMin</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="rangeDefault">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="minimum">
          <number>-512</number>
         </property>
         <property name="maximum">
          <number>511</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="rangeMax">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="minimum">
          <number>-512</number>
         </property>
         <property name="maximum">
          <number>511</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ReachStringPicker</class>
   <extends>QWidget</extends>
   <header>../widgets/string_picker.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>listOptions</tabstop>
  <tabstop>buttonOptionsNew</tabstop>
  <tabstop>buttonOptionsMoveUp</tabstop>
  <tabstop>buttonOptionsMoveDown</tabstop>
  <tabstop>buttonOptionsDelete</tabstop>
  <tabstop>optionType</tabstop>
  <tabstop>listValues</tabstop>
  <tabstop>buttonValuesNew</tabstop>
  <tabstop>buttonValuesMoveUp</tabstop>
  <tabstop>buttonValuesMoveDown</tabstop>
  <tabstop>buttonValuesDelete</tabstop>
  <tabstop>valueValue</tabstop>
  <tabstop>rangeMin</tabstop>
  <tabstop>rangeMax</tabstop>
  <tabstop>rangeDefault</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
